src/hooks/standardLibs.ts

```diff
@@ -0,0 +1,152 @@
+import assert from "node:assert";
+
+import { ESTree } from "meriyah";
```

Comment:
Please try to use `import type` when applicable to make it clear when code is being imported vs. just types.